packages/core/src/components/document-component-demo/document-component.demo.tsx

```diff
@@ -275,6 +300,13 @@ export class DocumentComponentDemo implements ComponentInterface {
     );
   }
 
+  private showInTableOfContents(type?: string): boolean {
+    return (
+      !!type &&
+      ["LICHAAM", "HOOFDSTUK", "AFDELING", "ARTIKEL", "PARAGRAAF", "SUBPARAGRAAF", "SUBSUBPARAGRAAF"].includes(type)
```

Comment:
Moet ik DIVISIE en DIVISIETEKST ook opnemen?

Of kan ik het beter omdraaien en er type !=='LID' van maken?

Reply:
Ja, ik kan het beter omdraaien.